shardingsphere-test/shardingsphere-parser-test/src/main/resources/case/ddl/alter-foreign-data-wrapper.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to You under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<sql-parser-test-cases>
+    <alter-foreign-data-wrapper sql-case-id="alter_foreign_data_wrapper_case1" />
+    <alter-foreign-data-wrapper sql-case-id="alter_foreign_data_wrapper_case2" />
+    <alter-foreign-data-wrapper sql-case-id="alter_foreign_data_wrapper_case3" />
+    <alter-foreign-data-wrapper sql-case-id="alter_foreign_data_wrapper_case4" />
+    <alter-foreign-data-wrapper sql-case-id="alter_foreign_data_wrapper_case5" />
+    <alter-foreign-data-wrapper sql-case-id="alter_foreign_data_wrapper_case6" />
```

Comment:
@tuichenchuxin Can we rename `sql-case-id` with more meaningful name?

Reply:
ok,done